packages/itwinui-css/src/utils/icon/classes.scss

```
@@ -1,23 +1,26 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
-@import './index';
+@use './icon-component.scss' as *;
 
-[data-iui-icon-color='default'] {
-  @include iui-icon-color;
-}
-
-[data-iui-icon-color='informational'] {
-  @include iui-icon-color(informational);
+.iui-svg-icon,
+[data-iui-icon-color] {
```

If having `data-iui-icon-color` without `.iui-svg-icon` is treated like an icon, should having `data-iui-icon-size` without `.iui-svg-icon` also be treated like an icon?

```suggestion
.iui-svg-icon,
[data-iui-icon-color], {
[data-iui-icon-size], {
```

---

no need, because that was done for backwards compatibility whereas `data-iui-icon-size` is a new addition